src/includes/before-breadcrumb/android.mdx

```diff
@@ -18,6 +18,21 @@ SentryAndroid.init(options -> {
 });
 ```
 
+```kotlin
+import io.sentry.android.core.SentryAndroid
+import io.sentry.SentryOptions.BeforeBreadcrumbCallback
+
+SentryAndroid.init { options ->
```

Comment:
```suggestion
SentryAndroid.init(this) { options ->
```

every `SentryAndroid.init` call requires the `this` param.
so maybe a string replace would be easier than commenting 1 by 1 :D

Reply:
Updated. `this` was missing in quite few Android-Java samples too.

Reply:
good catch, thanks for doing that